docs/source/install.rst

```diff
@@ -130,7 +130,12 @@ the installation of a particular CUDA version (say 10.0) for driver compatibilit
 
 .. note::
 
-    Currently cuTENSOR is not yet available on ``conda-forge``.
+    cuTENSOR is available on ``conda-forge`` for CUDA 10.1+ and is an optional dependency. To install CuPy with the cuTENSOR
+    support enabled, you can do::
+
+    $ conda install -c conda-forge cupy cutensor cudatoolkit=10.2
+
+    ``cupy`` and ``cutensor`` must coexist in order for the ``conda`` solver to pick the right package.
```

Comment:
Does "coexist" mean "needs to be installed in a single command"?
Sounds a bit ambiguous to me.

Reply:
Indeed. How about "CuPy and cutensor must be installed at the same time"?

Reply:
Sounds good to me!

Reply:
Thanks! Done in 866ab84.